packages/core/src/navigation/navigation-start.element.ts

```diff
@@ -18,7 +18,6 @@ import {
   state,
 } from '@cds/core/internal';
 import itemStyles from './navigation-item.element.scss';
-import styles from './navigation-start.element.scss';
```

Comment:
Removed file, since it was empty it was logging unnecessary warnings out from rollup.